### ISSUE TITLE
DB Port

### ISSUE THREAD

**Issue by AnthoDingo** [hardware/postfixadmin#9]
DBPORT no working with postfixadmin.
Your changes set connection uri to "$server:$host"
Howewer, postfix admin use mysqli connection : 

```php
function db_connect ($ignore_errors = 0) {
...
} elseif ($CONF['database_type'] == "mysqli") {
        if (function_exists ("mysqli_connect")) {
            $link = @mysqli_connect ($CONF['database_host'], $CONF['database_user'], $CONF['database_password']) or $error_text .= ("<p />DEBUG INFORMATION:<br />Connect: " .  mysqli_connect_error () . "$DEBUG_TEXT");
            if ($link) {
                @mysqli_query($link,"SET CHARACTER SET utf8");
                @mysqli_query($link,"SET COLLATION_CONNECTION='utf8_general_ci'");
                @mysqli_select_db ($link, $CONF['database_name']) or $error_text .= ("<p />DEBUG INFORMATION:<br />MySQLi Select Database: " .  mysqli_error ($link) . "$DEBUG_TEXT");
            }
        } else {
            $error_text .= "<p />DEBUG INFORMATION:<br />MySQL 4.1 functions not available! (php5-mysqli installed?)<br />database_type = 'mysqli' in config.inc.php, are you using a different database? $DEBUG_TEXT";
        }
...
}
```

Mysqli constructor :
http://php.net/manual/fr/mysqli.construct.php
($host, $username, $password, $dbname, $port)

Searching solution without editing postfix original code. Maybe submit on the original project.

**Comment by michael-k** [comment]
Please put ` ```php ` at the beginning and ` ``` ` at the end of your code examples.

Looks like this then:
```php
function db_connect ($ignore_errors = 0) {
    // code
}
```

**Comment by hardware** [comment]
Database port option for mysqli connection has been added in september 2017 but this change is not yet available in the latest version (3.1) :

https://github.com/postfixadmin/postfixadmin/commit/4b999b3f6bb78f595af2a7294f6b231de8f61fc4

**Comment by hardware** [comment]
You can now use `DBPORT` variable.
